docs/SupportedONNXOps-cpu.md

```diff
@@ -37,7 +37,7 @@ Onnx-mlir currently supports ONNX operations targeting up to opset 19. Limitatio
 | **BitwiseXor** |18 - * | | |
 | **BlackmanWindow** |none | | | |
 | **Cast** |6 - 18 |Cast only between float and double types. Only ppc64le and MacOS platforms support float16. | |
-| **CastLike** |none | | | |
+| **CastLike** |19 - * |CastLike only between float and double types. Only ppc64le and MacOS platforms support float16. | |
```

Comment:
x86? Personally don't know

Reply:
@AlexandreEichenberger I copied the same thing that `Cast` had... look at line 39. So should be the same...tests are the same as well. So I decided to add the same comment :)